komodor/datasource_komodor_kubernetes.go

```diff
@@ -0,0 +1,39 @@
+package komodor
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceKomodorKubernetes() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceKomodorKubernetesRead,
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The id and api key of the cluster integration",
+			},
+			"cluster_name": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Please add a description, It is worth mentioning that the cluster name should be unique in the account

Reply:
Description added.